static/js/crit.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('DOMContentLoaded', function(event){
+
+    document.getElementsById("1_crit").addEventListener('click', function(event){
```

Comment:
This javascript code is specific to one single element on the page (potentially). Consider removing until you're ready to write a client-side JS click handler for this.